File: implementation/scipamato/core/web/src/test/java/ch/difty/scipamato/common/UtilTestConfiguration.java
package ch.difty.scipamato.common;

import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;
import org.springframework.context.annotation.Primary;

@Configuration
public class UtilTestConfiguration {

    @Bean
    @Primary
    public DateTimeService dateTimeService() {
        return new FrozenDateTimeService();
    }
}

File: core/core-web/src/test/kotlin/ch/difty/scipamato/common/UtilTestConfiguration.kt
package ch.difty.scipamato.common

import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration
import org.springframework.context.annotation.Primary

@Configuration
open class UtilTestConfiguration {

    @Bean
    @Primary
    open fun dateTimeService(): DateTimeService = FrozenDateTimeService()
}
